docs/source/user/api.rst

```diff
@@ -385,7 +398,7 @@ Return also the radius user token in the response.
 
 .. code-block:: text
 
-    /api/v1/account/token/validate/
+    /api/v1/<organization-slug>account/token/validate/
```

Comment:
Not, I noticed this did not include `<organization-slug>` but in usage, I was putting <org-slug> to get the token validation, I assume this one only a typing mistake.